### PR TITLE
Add editorial blockquote.

### PR DESCRIPTION
**Pull request by notlee**
This commit is a big one which was difficult to break up:

Updates the README with a Sass section, contact section, etc.

Renames private Sass placeholders to remove BEM syntax as they
no longer map directly to a class of the same name.

Deprecates `oQuoteBase` and `oQuoteBaseCite`. Users are able to
use `oQuoteStandard` and `oQuoteStandardCite` alone, or the new
editorial variant equivalents, without remembering to also include
the base mixins.

The base o-quote styles are stripped back so we don't have to
undo these styles in the future when we add other variants
such as a pull quote -- I'm not quite sure why base styles
exist at all, they don't seem to be used.

Maintains the current standard style for use in non-editorial
contexts, this may be a candidate for deprecation in the near future.

Adds a new `o-quote--editorial` modifier and associated Mixins.
Going forward this should be used in article content over the
standard version which is currently used on ft.com:
https://github.com/Financial-Times/n-content-body/blob/c703b86dd04561f240b2e6cb08f0472eba9566a2/scss/_blockquotes.scss#L5

### Base
Before:
Screenshot 2020-04-07 at 16 40 06

After:
Screenshot 2020-04-07 at 16 38 55

### Standard

Before: 
Screenshot 2020-04-07 at 16 40 08

After:
Screenshot 2020-04-07 at 16 38 58

### Editorial (new)

Screenshot 2020-04-07 at 16 39 01

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,94 @@
...
+- [Markup](#markup)
+- [Sass](#sass)
+- [Migration guide](#migration-guide)
+- [Contact](#contact)
+- [Licence](#licence)
 
 ## Markup
 
-Add an `o-quote` class to any quote you wish to apply the styles to.
+For an standard blockquote style use classes `o-quote o-quote--standard` and the following markup:
```
Comment (JakeChampion) [inline README.md:13]:
```suggestion
For a standard blockquote style use classes `o-quote o-quote--standard` and the following markup:
```

File: bower.json [hunk 2]
```diff
@@ -11,6 +11,7 @@
     "o-colors": "^5.0.0",
     "o-icons": "^6.0.0",
     "o-typography": "^6.0.0",
+    "o-editorial-typography": "^1.0.7",
```
Comment (JakeChampion) [inline bower.json:14]: Is this now a major version bump because of this dependency addition?

File: main.scss [hunk 3]
```diff
@@ -22,27 +24,55 @@ $_o-quote-first-import: true !default;
...
+	// Enabled if the option is given and the current brand supports the feature.
+	$standard-enabled: map-get($opts, 'standard') and _oQuoteSupports('standard');
+	$editorial-enabled: map-get($opts, 'editorial') and _oQuoteSupports('editorial');
 	$cite-enabled: map-get($opts, 'cite');
-	$standard-enabled: map-get($opts, 'standard');
 
+	// Output base quote styles.
 	.o-quote {
-		@include oQuoteBase();
+		@extend %_o-quote;
```
Comment (JakeChampion) [inline main.scss:41]: Thinking out loud: I hardly see Origami styles using `@extend`.

File: bower.json (hunk 2 above)
Comment (notlee) [reply bower.json:14]: I don't think we should consider this major as there is only one version, so no conflicts.

File: main.scss (hunk 3 above)
Comment (notlee) [reply main.scss:41]: In the past we have strongly discouraged them. They can lead to an unpredictable source order and previously caused us problems particularly as placeholders would be output multiple times, whenever a component was imported. We avoid most issues by only outputting placeholders once with a flag and keeping them private behind mixins. o-quote is a bit of an experiment to see if there's anything else we haven't considered!

File: bower.json (hunk 2 above)
Comment (JakeChampion) [reply bower.json:14]: Excellent

**Comment by JakeChampion** [comment]
@notlee releastetype:minor?

**Pull request merged**